client/components/learn/Resources/resources-model.ts

```diff
@@ -41,6 +41,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://validator.w3.org/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/w3c-validator_500w.png`,
     },
+    {
+      title: 'Patterns Dev',
+      url: 'https://patterns.dev/',
+      cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/patterns.dev.png`,
```

Comment:
Please resize the asset to `500px` width. Then rename to `patters-dev_500w.png` (notice the `_500w` at the end, it's the same as all other assets).